src/main/resources/db_scripts/scheduled/archive-users-transaction.sql

```diff
@@ -0,0 +1,36 @@
+-- This is NOT scheduled via Quartz, but intended to be run regularly using cron.
```

Comment:
Out of curiosity, why not?

Reply:
Partly a separation of concerns, but mostly so that we have scope for moving the ownership of the archived table to a new Postgres user in the future. If we did that, the API would be unable to read or write it and so couldn't access it via Quartz, which would then protect the archival data from a whole class of vulnerabilities (SQL injection) in the web server.

That was too complex to build right away, but could be dropped in later without changing this script.